modules/Layout/CookieConsentBar/CookieConsentBar.module.scss

```diff
@@ -1,6 +1,8 @@
+@import "variables";
```

Comment:
No need to import anything in SCSS modules, everything needed is injected via the SCSS config.

Reply:
Yeah, you're right that it does work. My IDE warns about missing imports though. I'll remove the imports and will just ignore the warnings. Thanks!

![Screenshot from 2022-08-10 11-41-56](https://user-images.githubusercontent.com/370680/183856804-c890d40a-41c4-4b06-9973-c147bc0e1662.png)